benchmark/framework/aea_test_wrapper.py

```diff
@@ -168,15 +167,15 @@ def set_loop_timeout(self, timeout: float) -> None:
 
         :return: None
         """
-        self.aea._timeout = timeout
+        self.aea._timeout = timeout  # pylint: disable=protected-access
```

Comment:
setter? Although we shouldn't allow undesired change of timeout... So probably better leave it for now.

Reply:
Yes I think it should be left as is (this is a unique exception imo)